src/components/ContentHighlights/ContentHighlightsCardItemsContainer.jsx

```diff
@@ -21,19 +17,43 @@ const ContentHighlightsCardItemsContainer = () => {
         xl: 4,
```

Comment:
nit: should we align to using the same `columnSizes` from other components here? I.e., 

```jsx
columnSizes={{
  xs: 12,
  md: 6,
  lg: 4,
  xl: 3,
}}
```

[suggestion] maybe even moving this to a constant so that we can ensure the `CardGrids`  throughout all of `ContentHighlights` all have the same grid layout config?